molecule/test-cluster/playbook.yml

```diff
@@ -6,17 +6,17 @@
   vars:
     ansible_python_interpreter: '{{ ansible_playbook_python }}'
     deploy_dir: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/deploy"
-    image_name: api.astarte-project.org/astarte-operator:testing
+    image_name: api.astarte-platform.org/astarte-operator:testing
```

Comment:
While we're at this: image_name should be astarte/astarte-kubernetes-operator:latest